### ISSUE TITLE
Announce ‘No results found for your search’ to screen reader users.

### ISSUE THREAD

**Issue by mark-dce** [yalelibrary/YUL-DC#828]
**DETAILS**
See issue 12 in https://drive.google.com/open?id=1KsZmdBi00ScskmAMffnM1O_N-CnMUZIv

**ACCEPTANCE**
- [ ] Screen reader users get an appropriate announcement when no search results are returned

**Comment by mikeapp** [comment]
Maybe a breadcrumb? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by dl-maura** [comment]
https://github.com/samvera-deprecated/sufia/issues/1585

**Comment by orangewolf** [comment]
@mark-dce these reload the page, they do not appear to be turbo-links loads. They also appear to show up in the screen reader, so I'm not clear what is missing here. I'm not an accessibility expert, so I 100% believe that something may need to be done, but this is what I'm seeing with Voice Over and therefor could use some more clarification as to what we should accomplish.  I'm happy to hop on with Kara if that's the right thing to do. Let me know if I should email here directly or if there is a better way to set that up.

**Comment by mark-dce** [comment]
It seems like the next step would be to get together with Kara and compare notes.

**Comment by dl-maura** [comment]
I've emailed Kara about this, waiting to hear back

**Comment by dl-maura** [comment]
From Kara:

Currently, a screen reader user will not hear any feedback after they submit a search. The page refreshes, and they need to find the results of their search manually (by using the screen reader rotor to find the heading or by jumping down to the Search Results section). This is not an ideal user experience, as it requires a screen reader user to have to find the outcome of their search. Whereas visual users can see the results right away.
 
A better solution would be to announce the search results status. This will provide instant user feedback for screen reader users. For example, on gov.uk/search the results are announced after search submit (using aria-live).

Further reading about Status Messages from W3: https://www.w3.org/WAI/WCAG21/Understanding/status-messages

**Comment by orangewolf** [comment]
https://github.com/yalelibrary/yul-dc-blacklight/pull/397

**Comment by alishaevn** [comment]
verified that the voice over announces when there are no results and when there are results.

no results: https://p-mYF314A.t4.n0.cdn.getcloudapp.com/items/9ZuBwLn1/Screen%20Recording%202020-11-17%20at%2012.01.46%20PM.mp4

results: https://p-mYF314A.t4.n0.cdn.getcloudapp.com/items/GGurOyOb/Screen%20Recording%202020-11-17%20at%2012.04.38%20PM.mp4
